restapi/src/main/java/io/stargate/web/docsapi/service/query/filter/operation/FilterHintCode.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.web.docsapi.service.query.filter.operation;
+
+import io.stargate.web.docsapi.service.query.FilterExpression;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Optional;
+
+/** Enumerates tags for query hints. */
+public enum FilterHintCode {
+
+  /**
+   * Filter selectivity.
+   *
+   * @see FilterExpression#getSelectivity()
+   */
+  SELECTIVITY("$selectivity");
```

Comment:
Can we get this added to the swagger in https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java#L292 and create an issue in stargate/docs to make sure full documentation doesn't get lost?

Reply:
sure. Sorry, I missed that. Follow-up PR: #1138

Reply:
Docs issue: https://github.com/stargate/docs/issues/144